<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeformableRegistrationClass</class>
 <widget class="QMainWindow" name="DeformableRegistrationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeformableRegistration</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="Viewer" name="view" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpenTemplate"/>
    <addaction name="actionOpenTarget"/>
    <addaction name="separator"/>
    <addaction name="actionSaveTemplate"/>
    <addaction name="actionSaveTarget"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHaoLi2008">
    <property name="title">
     <string>Hao Li 2008</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHaoLi2008"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpenTemplate">
   <property name="text">
    <string>Open Template...</string>
   </property>
  </action>
  <action name="actionOpenTarget">
   <property name="text">
    <string>Open Target...</string>
   </property>
  </action>
  <action name="actionSaveTemplate">
   <property name="text">
    <string>Save Template...</string>
   </property>
  </action>
  <action name="actionSaveTarget">
   <property name="text">
    <string>Save Target...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>Viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DeformableRegistration.qrc"/>
 </resources>
 <connections/>
</ui>
